### PR TITLE
Synthesis added

### PR DESCRIPTION
**Pull request by ravstar52**
So. Many. Recipes.

addresses #81

### CODE REVIEW

File: EDEngineer/Resources/Data/blueprints.json [hunk 1]
```diff
@@ -13545,5 +13545,713 @@
...
+      {
+        "Name": "Arsenic",
+        "Size": 1
+      },
+      {
+        "Name": "Niobium"
+        "Size": 3
+      },
+      {
+        "Name": "Yttrium"
```
Comment (msarilar) [inline EDEngineer/Resources/Data/blueprints.json]: missing comma at the end

File: EDEngineer/Resources/Data/blueprints.json [hunk 2]
```diff
@@ -13545,5 +13545,713 @@
...
+        "Size": 1
+      },
+      {
+        "Name": "Polonium"
```
Comment (msarilar) [inline EDEngineer/Resources/Data/blueprints.json]: missing comma at the end

**Comment by msarilar** [comment]
(in case you missed that, I'll merge after the json becomes valid http://jsonlint.com/ ; currently the missing commas break the format)

**Comment by ravstar52** [comment]
Not a problem

**Pull request merged**